libtsuba/include/tsuba/RDGSlice.h

```diff
@@ -68,6 +70,7 @@ class KATANA_EXPORT RDGSlice {
   // Data
   //
 
+  tsuba::RDGTopology* topology_;
```

Comment:
The way I see it, there is really no reason RDGCore should have to be tied to some specific topology. RDGCore doesn't need it, and RDG doesn't need it (right now RDG::DoMake loads/maps the main CSR topology, but nothing actually depends on it being loaded so I'm toying with removing the mapping step altogether)

I believe RDGSlice only actually needs this pointer for one function call, so it is very possible we could remove it altogether. I just didn't dig enough into it yet since my focus was on other parts of the code.

If all of this makes sense, I'll add a TODO to clean this pointer up.

Reply:
That does all make sense. Thank you.